src/tests/group.controller.test.ts

```diff
@@ -464,6 +464,352 @@ describe('testing group actions', () => {
+        expect(res.body.users[0].invitedGroups).to.include(groupId);
+        done();
+      });
+  });
+
+  it('should get the second invited friend user object via GET /users/{userId}', done => {
+    chai
+      .request(server)
+      .get('/users/')
+      .query({ userId: userIdFriend2 })
+      .then(res => {
+        expect(res).to.have.status(200);
+        expect(res.body.users[0]).to.have.keys(USER_KEYS_TEST);
+        expect(res.body.users[0].invitedGroups).to.include(groupId);
+        done();
+      });
+  });
+
+  it('should get the third invited friend user object via GET /users/{userId} (friend5)', done => {
+    chai
+      .request(server)
+      .get('/users/')
+      .query({ userId: userIdFriend5 })
+      .then(res => {
+        expect(res).to.have.status(200);
+        expect(res.body.users[0]).to.have.keys(USER_KEYS_TEST);
+        expect(res.body.users[0].invitedGroups).to.include(groupId);
+        done();
+      });
+  });
+
+  it('should invite a new member to a group via PATCH /group/{groupId}/inviteMembers (friend3)', done => {
+    chai
+      .request(server)
+      .patch(`/groups/${groupId}/inviteMembers`)
+      .query({ userId: userIdMain })
+      .send([userIdFriend3])
+      .then(res => {
+        expect(res).to.have.status(200);
+        done();
+      });
+  });
+
+  it('should invite a new member to a group via PATCH /group/{groupId}/inviteMembers (friend6)', done => {
+    chai
+      .request(server)
+      .patch(`/groups/${groupId}/inviteMembers`)
+      .query({ userId: userIdMain })
+      .send([userIdFriend6])
+      .then(res => {
+        expect(res).to.have.status(200);
+        done();
+      });
+  });
+
+  it('should get a group by id via GET /group/{groupId}', done => {
+    chai
+      .request(server)
+      .get('/groups/')
+      .query({ groupId: groupId })
+      .then(res => {
+        expect(res).to.have.status(200);
+        expect(res.body.group).to.have.keys(GROUP_KEYS_TEST);
+        expect(res.body.group.members).to.include(userIdMain);
+        expect(res.body.group.invitedMembers).to.include(userIdFriend1);
+        expect(res.body.group.invitedMembers).to.include(userIdFriend2);
+        expect(res.body.group.invitedMembers).to.include(userIdFriend3);
+        expect(res.body.group.invitedMembers).to.not.include(userIdFriend4);
+        expect(res.body.group.invitedMembers).to.include(userIdFriend5);
+        expect(res.body.group.invitedMembers).to.include(userIdFriend6);
+        done();
+      });
+  });
+
+  it('should accept group invitation and add member to group (userIdFriend5)', done => {
+    chai
+      .request(server)
+      .patch(`/users/${userIdFriend1}/acceptGroupInvitation`)
+      .query({ groupId: groupId })
+      .then(res => {
+        expect(res).to.have.status(200);
+        done();
+      });
+  });
+
+  it('should accept group invitation and add member to group (userIdFriend5)', done => {
```

Comment:
same as above. 
`it(...)` refers to `userIdFriend5` but test is using `userIdFriend2`

```suggestion
  it('should accept group invitation and add member to group (userIdFriend2)', done => {
```